server/services/prisonerService.test.ts

```diff
@@ -0,0 +1,65 @@
+import nock from 'nock'
+import HmppsAuthClient from '../data/hmppsAuthClient'
+import config from '../config'
+import PrisonerService from './prisonerService'
+import { PrisonApiPrisoner } from '../@types/prisonApi/prisonClientTypes'
+import FullPageErrorType from '../model/FullPageErrorType'
+
+jest.mock('../data/hmppsAuthClient')
+
+const prisonerDetails = {
+  offenderNo: 'A1234AA',
+  firstName: 'Anon',
+  lastName: 'Nobody',
+  latestLocationId: 'LEI',
+  locationDescription: 'Inside - Leeds HMP',
+  dateOfBirth: '24/06/2000',
+  age: 21,
+  activeFlag: true,
+  legalStatus: 'REMAND',
+  category: 'Cat C',
+  imprisonmentStatus: 'LIFE',
+  imprisonmentStatusDescription: 'Serving Life Imprisonment',
+  religion: 'Christian',
+  agencyId: 'MDI',
+} as PrisonApiPrisoner
+
+const token = 'token'
+
+describe('Prisoner service related tests', () => {
+  let hmppsAuthClient: jest.Mocked<HmppsAuthClient>
+  let prisonerService: PrisonerService
+  let fakeApi: nock.Scope
+  beforeEach(() => {
+    config.apis.prisonApi.url = 'http://localhost:8100'
+    fakeApi = nock(config.apis.prisonApi.url)
+    hmppsAuthClient = new HmppsAuthClient(null) as jest.Mocked<HmppsAuthClient>
+    prisonerService = new PrisonerService(hmppsAuthClient)
+  })
+  afterEach(() => {
+    nock.cleanAll()
+  })
+
+  describe('prisonerService', () => {
```

Comment:
Is this one necessary?

Reply:
No its not. This was copied from the prisoner service in CRD. I'll address it in the next one